examples/tests/main.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
```

Comment:
Unfortunately the TF framework docgen stuff appears ultra simplistic, I'd love to get these examples wired up into test scripts or stuffed into the provider docs somehow at some point

Reply:
amen to that...